sourcewrap/blank.go

```diff
@@ -10,11 +10,15 @@ import (
 )
 
 // Blank operates as a blank source in its default state, but has a SetSource
-// method for setting the source later, it then uses the dials.Watcher
-// interface to update the View it's inserted into.
+// method for setting the source later, it then uses the dials.Watcher interface
+// to update the View it's inserted into.
 //
 // Blanks cannot be reused as they have to be aware of parameters of a
 // particular View.
+//
+// Note that when using this, consider setting the `SkipInitialVerification` to
```

Comment:
Either drop `the` before `SkipInititialVerification`, or add a qualifier (`field` or `option`).